packages/pigment-css-react/README.md

```diff
@@ -854,3 +854,216 @@ function App() {
   )
 }
 ```
+
+## Building a UI library
+
+The key of a UI library is to provide a set of reusable components that can be used and themed across **different projects**.
+
+This guide will show you how to build a statistics component as part of a UI library using Pigment CSS. There are 3 steps to follow:
```

Comment:
```suggestion
This guide will show you how to build a KPI card component as part of a UI library using Pigment CSS. There are 3 steps to follow:
```

Trying to find a better name for this component 😃 Just a quick suggestion.

Reply:
Sounds good to me.